docs/docs-beta/docs/guides/data-modeling/data-assets.md

```diff
@@ -2,3 +2,78 @@
 title: "Create data assets"
 sidebar_position: 10
 ---
+
+Data assets are at the core of Dagster. By modeling your data as assets, you unlock: Declarative Automation, lineage, and a searchable data catalog. This guide will help you understand how to model your data as assets.
+
+There are four ways to create data assets:
+
+* Using the `@asset` decorator to declare a single asset.
+* Using the `@multi_asset` decorator to output multiple assets from a single operation.
+* Using the `@graph_asset` decorator to output a single asset from multiple operations without making each operation itself an asset.
+* Using the `@graph_multi_asset` decorator to output multiple assets from multiple operations
+
+This guide walks you the first three methods. The fourth combines the approaches of the 2nd and 3rd so it's omitted.
+
+***
+
+<details>
+  <summary>Prerequisites</summary>
+
+To follow the steps in this guide, you'll need:
+
+* Dagster installed and a working project setup
+
+</details>
+
+***
+
+## Define a single data asset using the `@asset` decorator
+
+The simplest way to define a data asset in Dagster is by using the `@asset` decorator. This decorator marks a Python function as an asset, with the function's output being the data produced by the asset.
+
+<CodeExample filePath="guides/data-assets/data-assets/asset_docorator.py" language="python" title="Using @dg.asset decorator" />
```

Comment:
I think this code example should include two assets with a dependency between them (using `deps`) -- the dependency bit pretty essential to the whole asset system